src/System.IO.Pipelines/ref/System.IO.Pipelines.csproj

```diff
@@ -5,6 +5,9 @@
     <!-- We only plan to use this ref in netcoreapp. For all other netstandard compatible frameworks we should use the lib
     asset instead. -->
     <PackageTargetFramework Condition="'$(TargetGroup)' == 'netstandard'">netcoreapp2.0</PackageTargetFramework>
+    <!-- Must match version supported by frameworks which support 4.0.2.0 inbox.
+         Can be removed when API is added and this assembly is versioned to 4.1.* -->
+    <AssemblyVersion Condition="'$(TargetsNetFx)' != 'true'">4.0.2.0</AssemblyVersion>
```

Comment:
I'm not sure about this. I copied it from the [Determine AssemblyVersion section](https://github.com/dotnet/runtime/blob/0515878d1eb4fb544ae0219c436c299746fdb2b4/docs/project/library-servicing.md#determine-assemblyversion) of the library servicing guide.

I think it makes sense since System.IO.Pipelines dll ships as part of the Microsoft.AspNetCore.App shared runtime, but it's not directly part of the Microsoft.NETCore.App runtime, so I don't think I should list "netcoreapp3.0" et al. to the InboxOn section in packageIndex.json or add an InboxOnTargetFramework item to the pkgproj.

I'm not sure what the `'$(TargetsNetFx)' != 'true'` does. I hope this overrides the 4.0.2.1 AssemblyVersion configured in Directory.Build.props when building the library for the ASP.NET Core shared runtime, and does nothing when building the library for the NuGet package. @Anipik Is this right?